### ISSUE TITLE
Can't knit any of the template files with pandoc version ‘2.3.1’.

### ISSUE THREAD

**Issue by ahoffrichter** [mitchelloharawild/vitae#141]
Hello! This looks like a pretty cool package to make nice CVs. 

I couldn't install the package from CRAN, that's why I installed the development version. That worked fine. 

Unfortunately I cannot knit any of the template files, even without changing anything within them. 
I get different error messages/behaviour for the different templates. 

For `output: vitae::awesomecv` I get the following error: 

```Error running filter /var/folders/2x/pqn42dhx1392f4w2zfkjg0j80000gn/T//RtmpacLpGI/multiple-bibliographies.lua:
[string "/var/folders/2x/pqn42dhx1392f4w2zfkjg0j80000g..."]:34: attempt to compare string with table
Error: pandoc document conversion failed with error 83
Execution halted
```
For `output: 
  vitae::latexcv:
    theme: classic`, `output: vitae::twentyseconds` and `output: vitae::moderncv`

the same as above and: 

```
In addition: Warning message:
In utils::citation(..., lib.loc = lib.loc) :
  no date field in DESCRIPTION file of package 'vitae'
```

and for `output: vitae::hyndman` 

it just gets stuck at some point after processing already all chunks of the Rmd file. 

I'm not sure if it is really a problem with your package, or with something else (maybe knitr?). But maybe you are able to help me :)? 

Here is my `sessionInfo()`

```
R version 4.0.3 (2020-10-10)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Catalina 10.15.7

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] crayon_1.3.4     digest_0.6.27    dplyr_1.0.2      R6_2.5.0         lifecycle_0.2.0  magrittr_1.5    
 [7] evaluate_0.14    pillar_1.4.6     rlang_0.4.8      rstudioapi_0.11  ellipsis_0.3.1   vctrs_0.3.4     
[13] generics_0.1.0   rmarkdown_2.5    tools_4.0.3      glue_1.4.2       purrr_0.3.4      xfun_0.19       
[19] yaml_2.2.1       compiler_4.0.3   pkgconfig_2.0.3  htmltools_0.5.0  vitae_0.3.0      tidyselect_1.1.0
[25] knitr_1.30       tibble_3.0.4    
```

**Comment by mitchelloharawild** [comment]
Thanks for the bug report. This looks to be related to the rewrite of `bibliography_entries()` and its use of pandoc instead of biblatex.

Could you let me know which version of pandoc you are using?
``` r
rmarkdown::pandoc_version()
#> [1] '2.11.0.4'
```

<sup>Created on 2020-11-10 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by ahoffrichter** [comment]
I'm using ‘2.3.1’. 

I guess, that means, I should update pandoc?

**Comment by mitchelloharawild** [comment]
Not necessarily, although that will probably fix the problem.
It's useful for me to know which versions of pandoc are not working, so that I can fix the problem for others with version 2.3.1.

If you're using RStudio and would like to test if pandoc v2.11.0 fixes the issue, you can update pandoc by using the latest RStudio daily build: https://dailies.rstudio.com/

**Comment by ahoffrichter** [comment]
Thanks Mitchell! This is working for me perfectly!

**Comment by mitchelloharawild** [comment]
Great, thanks for checking!

**Comment by mpjashby** [comment]
> Not necessarily, although that will probably fix the problem.
> It's useful for me to know which versions of pandoc are not working, so that I can fix the problem for others with version 2.3.1.

For information, I had the same problem (rmarkdown hangs while knitting CVs using the Hyndman template) with pandoc 2.7.3 but updating to pandoc 2.11.2 (externally, not by updating RStudio) and it then works fine.

**Comment by mitchelloharawild** [comment]
I'm unable to reproduce this issue with pandoc v2.7.3.
Closing this issue as the latest release version of RStudio ships with pandoc v2.11.2.
